Jenkinsfile

```diff
@@ -34,7 +34,7 @@ def executionID = UUID.randomUUID().toString()
 def podName = 'fedora-cloud-' + executionID + '-allpkgs'
```

Comment:
When this PR is merged, won't your fork and this be the same? So we can go back to master?

Reply:
Yep, I just made that change to test a build quick.